src/table/pagination-table/components/head/ColumnAdjuster.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useRef } from 'react';
-import { AdjusterProps } from '../../types';
-import { useContextSelector, TableContext } from '../../context';
-import { AdjusterHitArea, AdjusterHeadBorder, AdjusterBodyBorder } from './styles';
-import { ColumnWidthTypes, MIN_COLUMN_WIDTH, PAGINATION_HEIGHT } from '../../constants';
-import { BORDER_WIDTH } from '../../styling-defaults';
+import { AdjusterProps } from '../../../types';
```

Comment:
the `ColumnAdjuster` will be used for the scrolling as well, so might as well move it now, the styles seem to be there already

Reply:
✅